src/lib/components/Checkbox.svelte

```diff
@@ -28,7 +30,8 @@
 <div
   tabindex="0"
   on:click|preventDefault={onClick}
-  on:keypress={($event) => handleKeyPress({ $event, callback: change })}
+  on:keypress={($event) =>
+    !disabled && handleKeyPress({ $event, callback: change })}
```

Comment:
I was wondering how a disabled checkbox would even get a keypress event because how can you focus a disabled checkbox?
So then I thought maybe the `tabindex` should be removed if the checkbox is disabled?

But feel free to consider this outside the scope of the PR.